views/web2py_ajax.html

```diff
@@ -9,7 +9,7 @@
     )}}
     //--></script>
 {{
-response.files.insert(0,URL('static','js/jquery.min.js'))
+response.files.insert(0,URL('static','js/jquery.js'))
```

Comment:
This looks like an unintentional commit

Reply:
It was just to test, as I hadn't minified the jquery lib, but I guess I can revert it now.